src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -53,14 +53,14 @@ Add your Dart code to the `lib/` directory.
 Add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter packages
 and plugins.
 
-The `.ios/` hidden subfolder contains a Xcode workspace where you can
-run a stand-alone version of your module, and is a wrapper project to bootstrap
-your Flutter code. It contains helper scripts to build frameworks or
+The `.ios/` hidden subfolder contains an Xcode workspace where you can
```

Comment:
I'll be damned, it _is_ "an", I've been getting that wrong for years... https://developer.apple.com/documentation/xcode/creating_an_xcode_project_for_an_app